packages/editor/.env.development

```diff
@@ -1,4 +1,4 @@
 PORT=3000
-HTTPS=false
+HTTPS=true
```

Comment:
This is not related to the changes here, but I have a question. Are you using a package to set env vars from this config file? I still see "set HTTPS=true" in the scripts so I take it this really only works on Windows right now...

Reply:
create-react-app is reading these vars in for us, and it works cross platform. 

The set  HTTPS=true is not needed